app/form_models/offender_sar_complaint_case_form.rb

```diff
@@ -77,7 +77,7 @@ def params_after_step_confirm_offender_sar(params)
   end
 
   def params_after_step_date_received(params)
-    if [nil, "standard", "ico"].include? params["complaint_type"]
+    if [nil, "standard"].include? object["complaint_type"]
```

Comment:
What is `object` in this context? Could we give it a less generic name?

Reply:
it is indeed confusing,  actually this module is included in `OffenderBaseDecorator`, the `object` is the way designed from Draper::Decorator (super super class of `OffenderBaseDecorator`),  it is designed as generic name by purpose I guess

Reply:
sorry for not explaining the `object,` the object is the actual activerecord model instance.  e.g. 
for `OffenderBaseDecorator`,  the object would be offender_sar case